### PR TITLE
This branch is to create basic structure for Navbar

### PR DESCRIPTION
**Pull request by AnitaDames**
## Create Navbar Structure

**Add:**

- create a Navbar with an ol 
- h2 with website logo initials

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,7 +8,17 @@
...
+
+    <div class="container">
+        <nav>
+        <ul>
+            <li><a href="#">Welcome</a></li>
+            <li><a href="#">Gallery</a></li>
+        </ul>
+       </nav>
+
+    </div>
```
Comment (steveWDamesJr) [inline index.html:21]: Good job implementing the basic nav structure 💯

File: index.css [hunk 2]
```diff
@@ -1,3 +1,7 @@
 h1{
     color: brown;
+}
+
+li {
+ list-style: none;
```
Comment (steveWDamesJr) [inline index.css:6]: Nice work eliminating bullet points from the nav links 🎉

File: index.html (hunk 1 above)
Comment (AnitaDames) [reply index.html:21]: Thank you 🤝

**Review by steveWDamesJr — APPROVED** [review]
Hi @TeacherAnita  , 

Your task is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

File: index.css (hunk 2 above)
Comment (AnitaDames) [reply index.css:6]: Thank you 🤝 

Happy Coding!

**Pull request merged**